src/lib/context/ThemeProvider.tsx

```
@@ -0,0 +1,127 @@
+// ThemeProvider.tsx
+'use client';
+
+import React, { createContext, useContext, useEffect, useState, type ReactNode } from 'react';
+import { ThemeProvider as NextThemesProvider, useTheme as useNextTheme } from 'next-themes';
+
+import {
+  ExperienceType,
+  ColorMode,
+  getTheme,
+} from '@/styles/theme';
+import { standardVariants, corporateVariants, getDefaultVariant, ThemeVariant } from '@/config/theme-variants';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'standardVariants'. Consider to delete it or rename it.

'standardVariants' is defined here:


(lint/suspicious/noRedeclare)